lib/inspec/cli.rb

```diff
@@ -39,12 +40,12 @@ class Inspec::InspecCLI < Inspec::BaseCLI
     desc: 'A list of controls to include. Ignore all other tests.'
   profile_options
   def json(target)
-    o = opts.dup
+    o = config
     diagnose(o)
     o['log_location'] = STDERR
```

Comment:
I am not sure but in some we are using key as symbols and here we are using string. Can we use symbol here too ?